Assets/HoloToolkit/SpectatorView/Images/PlatformSwitcher.PNG.meta

```diff
@@ -0,0 +1,76 @@
+fileFormatVersion: 2
```

Comment:
Is this image just for documentation?

If so, let's move it to where we put all the other documentation images for the project.